deployment/scripts/chainspec/beta_chainstate_spec.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Because the name of state and params is hardcoded, you need to create `beta` and `dev` folders that contain both files with the right name. We use this config directly for our dev and prod environments, so it should be valid.